libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/incident/Incident.kt

```diff
@@ -58,21 +59,19 @@ class IncidentAlert private constructor(
     /**
      * Use to create a new instance.
      *
-     * @see IncidentAlert
+     * @see Incident
      */
     class Builder(
-        private val coordinate: Point,
-        private val distance: Double
+        private val objectGeometry: RoadObjectGeometry
     ) {
-
-        private var alertGeometry: RouteAlertGeometry? = null
         private var info: IncidentInfo? = null
+        private var distanceFromStartOfRoute: Double? = null
 
         /**
-         * Add optional geometry if the alert has a length.
+         * Add optional distance from start of route.
          */
-        fun alertGeometry(alertGeometry: RouteAlertGeometry?): Builder = apply {
-            this.alertGeometry = alertGeometry
+        fun distanceFromStartOfRoute(distanceFromStartOfRoute: Double?): Builder = apply {
```

Comment:
the same https://github.com/mapbox/mapbox-navigation-android/pull/4109/files#r592430961. Also if it might be less than 0 could you mention it in the kdoc

Reply:
discussed above